src/factories/list/FactoriesKanban.tsx

```diff
@@ -0,0 +1,96 @@
+import { useStore } from '@/core/zustand.ts';
+import { useGameFactories } from '@/games/store/gameFactoriesSelectors.ts';
+import {
+  ControlledBoard,
+  KanbanBoard,
+  moveCard,
+} from '@caldwell619/react-kanban';
+import { Factory, FactoryProgressStatus } from '@/factories/Factory.ts';
+import { ProgressChip } from '@/factories/components/ProgressChip.tsx';
+import './FactoriesKanban.css';
+import { FactoryGridCard } from '@/factories/list/FactoryGridCard.tsx';
```

Comment:
in general, imports with extension are not necessary in this TS project. Please avoid them in this project, I'd probably need to add them to eslint

Reply:
Fixed!